modules/auxiliary/scanner/redis/redis_server.rb

```diff
@@ -0,0 +1,44 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+  include Msf::Auxiliary::Redis
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'         => 'Redis-server Scanner',
+      'Description'  => %q{
+          This module scans for Redis server. By default Redis has no auth. If auth
+        (password only) is used, it is then possible to execute a brute force attack on
+        the server. This scanner will find open or password protected Redis servers and
+        report back the server information
```

Comment:
This description no longer aligns with what the module now does.  Previously it would `PING` and then `INFO` and report the output of `INFO`, but now it just uses `PING`.

Perhaps a better way of handling this would be to make the command that it runs by default to identify a redis server  configurable as an advanced option.  Then, make the `report_service` report both the command used and the output.